conf/test_full.config

```diff
@@ -1,22 +1,24 @@
 /*
- * -------------------------------------------------
- *  Nextflow config file for running full-size tests
- * -------------------------------------------------
- * Defines bundled input files and everything required
- * to run a full size pipeline test. Use as follows:
- *   nextflow run nf-core/metapep -profile test_full,<docker/singularity>
- */
+========================================================================================
+    Nextflow config file for running full-size tests
+========================================================================================
+    Defines input files and everything required to run a full size pipeline test.
+
+    Use as follows:
+        nextflow run nf-core/metapep -profile test_full,<docker/singularity>
+
+----------------------------------------------------------------------------------------
+*/
 
 params {
-  config_profile_name = 'Full test profile'
-  config_profile_description = 'Full test dataset to check pipeline function'
+    config_profile_name        = 'Full test profile'
+    config_profile_description = 'Full test dataset to check pipeline function'
+
+    // Input data for full size test
+    // TODO nf-core: Specify the paths to your full test data ( on nf-core/test-datasets or directly in repositories, e.g. SRA)
+    // TODO nf-core: Give any required params for the test so that command line flags are not needed
+    input = 'https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/samplesheet/samplesheet_full_illumina_amplicon.csv'
```

Comment:
and here.

Reply:
There is no full test-dataset yet, can we postpone this?

Reply:
Sure, just use any test dataset, as you did already. This will be than used to run on release on AWS and displayed on the website as example output, see e.g. [ampliseq](https://nf-co.re/ampliseq/results#ampliseq/results-80b3cb8b05d3b596bd0a52866e7febe40ea497db/). Its fine for now.